community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -534,7 +534,7 @@ public void transactionRecovered( long txId )
             LegacyIndexApplierLookup legacyIndexApplierLookup =
                     dependencies.satisfyDependency( new LegacyIndexApplierLookup.Direct( legacyIndexProviderLookup ) );
 
-            boolean safeIdBuffering = FeatureToggles.flag( getClass(), "safeIdBuffering", false );
+            boolean safeIdBuffering = FeatureToggles.flag( getClass(), "safeIdBuffering", true );
```

Comment:
Feature toggles should be stored in `static final` fields to enable constant propagation optimisations, and also to avoid parsing strings and allocating objects all the time.